### ISSUE TITLE
nuget configuration is invalid

### ISSUE THREAD

**Issue by FamilyMan** [NuGet/Home#2669]
Just updated to the latest version. I now get 'nuget configuration is invalid.' when I go to Manage Nuget Packages

**Comment by rrelyea** [comment]
My guess is that you have a nuget.config that is invalid XML, which used to be silently ignored.
In 3.4.0 we changed that to error: https://github.com/NuGet/Home/issues/1567

**Comment by FamilyMan** [comment]
I don't know enough about the inner workings of NuGet to debug this. The NuGet.Config file that seems to be associated with my webforms app is in the path AppData\Roaming\JetBrains\ReSharperPlatformVs14\v05, because this is the NuGet.Config file that gets its time stamp updated when I open my project.

The file contains this:

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key=".NET Products Gallery" value="https://resharper-plugins.jetbrains.com/api/v2/curated-feeds/Wave_v5.0/" />
  </packageSources>
</configuration>
```

I can't see what is wrong with this XML.

**Comment by emgarten** [comment]
@FamilyMan check your machine wide configuration file at %appdata%/nuget/nuget.config

**Comment by FamilyMan** [comment]
I have a NuGet.Config file located in %appdata%/Roaming/NuGet/
It contains:

```
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
    <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
  </packageSources>
  <activePackageSource>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
  </activePackageSource>
</configuration>
```

I also have a directory %appdata%/.nuget/ (Note the . before nuget) that contains a directory /packages, but no NuGet.Config file

**Comment by rrelyea** [comment]
Is this website app hosted in IIS?
See: https://github.com/NuGet/Home/issues/2667

**Comment by FamilyMan** [comment]
Yes it is. It appears to be the same issue.

**Comment by emgarten** [comment]
@FamilyMan what type of project are you seeing this issue with? Does the project contain a packages.config or project.json file? Is there an msbuild project file?

**Comment by FamilyMan** [comment]
It's a Web Site. There is a packages.config which contains this:

```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="bootstrap" version="4.0.0-alpha2" targetFramework="net46" />
  <package id="d3" version="3.5.16" targetFramework="net46" />
  <package id="DocX" version="1.0.0.19" targetFramework="net46" />
  <package id="EntityFramework" version="5.0.0" targetFramework="net451" />
  <package id="jQuery" version="2.2.3" targetFramework="net461" />
  <package id="jQuery.Steps" version="1.1.0" targetFramework="net451" />
  <package id="jQuery.UI.Combined" version="1.11.4" targetFramework="net451" />
  <package id="jQuery.Validation" version="1.15.0" targetFramework="net46" />
  <package id="JSONEditor" version="4.2.1.3" targetFramework="net46" />
  <package id="Less.js" version="1.5.1" targetFramework="net451" />
  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.1" targetFramework="net46" />
  <package id="Microsoft.Net.Compilers" version="1.2.1" targetFramework="net46" developmentDependency="true" />
  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net46" />
  <package id="Select2.js" version="4.0.2" targetFramework="net46" />
  <package id="SharpDX" version="3.0.0-beta01" targetFramework="net46" />
  <package id="toastr" version="2.1.1" targetFramework="net451" />
</packages>
```

Under Documents/Visual Studio 2015/Project/localhost_50854 there is a .vs directory, a packages directory, a .sln file and a .sln.DotSetting.user file.

**Comment by zhili1208** [comment]
looks like this issue https://github.com/NuGet/Home/issues/2601, this is a web site project, right? is your nuget.exe version 3.4.3? I believe it's fixed in 3.4.3

**Comment by emgarten** [comment]
Thanks @FamilyMan that helps narrow it down, this looks like the same issue as https://github.com/NuGet/Home/issues/2667
